src/theme/components/Layout.tsx

```diff
@@ -61,20 +61,20 @@ const Layout = ({ children }: Props): JSX.Element => (
           enableDeclineButton={true}
           flipButtons={true}
           style={{
-            opacity: '90%',
+            background: '#000000',
           }}
           contentStyle={{
-            color: theme.colors.background,
+            color: '#FFFFFF',
           }}
           cookieValue={false}
           buttonStyle={{
             background: '#90EE90',
-            color: theme.colors.primary,
+            color: '#000000',
```

Comment:
❓ if we remove the usage of the theme colors, what is the point of defining them? are they used elsewhere?

Reply:
For the consent bar, we use specific colors, not used elsewhere.  
I don't see the interest to use the theme for this. 

  
:information\_source: More over, I will change the component of the Consent bar and its theme soon. So, I didn't want to spend more time on this topic :slightly\_smiling\_face:

Reply:
If we are changing the whole Consent bar, this is probably extra work to change the color here.
Anyway, I'll check the rendering to see the visual change which is what matters most here.